### PR TITLE
[Chore] CAFE3 Rename in Config Files

### PR DESCRIPTION
**Pull request by dgibso29**
### Pull Request Description
**When merged this pull request will:**
- Wholesale rename all references to `F3`, `F3_CA`, and `F3_CA_BUB` to their appropriate `CAFE` counterparts.
- Finish lingering requirements of #71 

### Release Notes
- Brewed a triple shot of CAFE

## IMPORTANT

- [x] The Release Notes section below must be filled out appropriately to explain the changes made in this PR. This section will be used in Framework Changelogs.
- [x] If the contribution affects [the wiki](https://github.com/Bubbus/F3_CA_BUB/wiki), please include your changes in this pull request so the wiki is consistently updated.
- [x] [Contribution Guidelines](https://github.com/Bubbus/F3_CA_BUB/blob/release/contributing.md) are read, understood and applied.
- [x] Title of this PR uses our standard template `[Descriptor] - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by dgibso29** [comment]
Successful dedi test with clients. No apparent issues with core systems.

File: configuration/briefings/part_briefing_west.sqf [hunk 1]
```diff
@@ -34,7 +34,7 @@ _cre = player createDiaryRecord ["diary", ["Credits","
 <br/>
 *** Insert mission credits here. ***
 <br/><br/>
-Made with F3 (http://www.ferstaberinde.com/f3/en/)
+Made with CAFE3 (https://github.com/CombinedArmsGaming/CAFE3/)
```
Comment (Poulern) [inline configuration/briefings/part_briefing_west.sqf]: I notice we are consistent with removing the last / at the end of URLs elsewhere in this framework 

Should it maybe be changed to (https://github.com/CombinedArmsGaming/CAFE3) to be consistent?

Comment (dgibso29) [reply configuration/briefings/part_briefing_west.sqf]: Good catch, thank you.

Comment (Poulern) [reply configuration/briefings/part_briefing_west.sqf]: Thank you for entertaining my pedantry, you made it extremely difficult to spot anything

Comment (dgibso29) [reply configuration/briefings/part_briefing_west.sqf]: 😳

File: configuration/logiVehicle.sqf [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*
 
     Logistics vehicle configuration
-    Inspired by F3_CA's logi vic system.  Initial work by Joecuronium, expanded by Bubbus.
+    Inspired by CAFE_CA's logi vic system.  Initial work by Joecuronium, expanded by Bubbus.
```
Comment (Poulern) [inline configuration/logiVehicle.sqf]: 👀

Comment (dgibso29) [reply configuration/logiVehicle.sqf]: Damn. Got me good.

**Review by Bubbus — APPROVED** [review]
Looks fine.  It's a wide-reaching refactor so I'd want to run some missions with this before we move it into `release`.  Let's get this in `develop` and run a couple midweeks/Saturdays with it.

**Pull request merged**